ui/components/app/currency-input/currency-input.js

```diff
@@ -56,14 +60,16 @@ export default function CurrencyInput({
     type,
   );
   const hideSecondary = !showFiat;
-  const primarySuffix = preferredCurrency || EtherDenomination.ETH;
+  const primarySuffix =
+    tokenSymbol || preferredCurrency || EtherDenomination.ETH;
   const secondarySuffix = secondaryCurrency.toUpperCase();
 
-  const [isSwapped, setSwapped] = useState(false);
   const [newHexValue, setNewHexValue] = useState(hexValue);
   const [shouldDisplayFiat, setShouldDisplayFiat] = useState(featureSecondary);
   const shouldUseFiat = hideSecondary ? false : Boolean(shouldDisplayFiat);
 
+  const isPrimary = !shouldUseFiat;
```

Comment:
Minor: I'd rename this variable to be more descriptive, or just replace usages of it with `!shouldUseFiat` for better readability

Reply:
Good call